careers/blocks/slider/slider.js

```diff
@@ -0,0 +1,218 @@
+import { buildEllipsis } from '../../scripts/scripts.js';
```

Comment:
@amol-anand Should we go full-CSS for this one and make the text clickable instead? So we could remove "buildEllipsis altogether".

Reply:
Done.